include/color_spinor.h

```diff
@@ -1192,6 +1192,42 @@ namespace quda {
     return y;
   }
 
+  /**
+     @brief Compute the matrix-vector product z = A * x + y
+     @param[in] A Input matrix
+     @param[in] x Input vector
+     @param[in] y Input vector
+     @return The vector A * x
+  */
+  template<typename Float, int Nc, int Ns> __device__ __host__ inline
+  ColorSpinor<Float,Nc,Ns> mv_add(const Matrix<complex<Float>,Nc> &A, const ColorSpinor<Float,Nc,Ns> &x, const ColorSpinor<Float,Nc,Ns> &y)
```

Comment:
Should we have a version that just does `A * x` (`mv_add(...A, ...x);`), just to have it handy (and maybe as a micro-optimization somewhere)?

Reply:
Addressed offline, it's indeed what `operator*` does